news/_posts/2020-12-09-pr-testing-on-prestashop.md

```diff
@@ -0,0 +1,173 @@
+---
+layout: post
+title:  "PR Testing on PrestaShop"
+subtitle: ""
+date:   2020-12-08 16:00:00
+authors: [ RobinFischer ]
+icon: icon-airplane
+tags:
+ - 
+---
+
+Prestashop created a Quality Assurance team 4 years ago, in order to improve the quality of the solution.
+For more than a year now, the team has grown quickly, allowing it to take quality to the next level ! The field of action is widening (test of new services like Prestashop Checkout, specification test and review…), but the core of the manual testers activity remains the same; testing the Pull Requests (“PR”) on Github, and testing every release.
+
+
+# I. Some questions you may have in mind
+
+### Why do you test every PR ?
+
+First question that might come to your mind : Why is there QA on each PR ? Why not testing only during build release ?
+
+- The earlier a bug is detected in the process, the easier it is to fix it
+- Test on each PR allows us to make more detailed tests, and to improve the overall quality of the PrestaShop solution
+- There can be several months between the merge of a PR and the first test of the release, enough time for the author of the PR to have “forgotten” what he did exactly.
+
+### How do you choose which PR should be tested first ?
+
+This is a legitimate question that pops regularly. It might be frustrating for a contributor to see his PR blocked in “Waiting for QA” for weeks, while other PR are validated and merged the same day they are created.
+
+There are some rules to rank  PRs. The QA team uses the following criterias (in order of importance) :
+
+1. The branch on which PR is based : the oldest branch has the highest priority. We will test 1.7.6.x PR before 1.7.7.x PR, and 1.7.7.x PR before Develop PR. Modules’s PR usually have the lowest priority.
```

Comment:
I think it's more about [Supported Branches](https://devdocs.prestashop.com/1.7/contribute/contribution-guidelines/supported-branches/) than "old" branches 😉  .

Reply:
Thanks, I added this precision in the article (with a link to the devdoc)